src/RenderWebGL.js

```diff
@@ -794,26 +800,22 @@ class RenderWebGL extends EventEmitter {
     }
 
     /**
-     * Detect which sprite, if any, is at the given location. This function will not
-     * pick drawables that are not visible or have ghost set all the way up.
+     * Detect which sprite, if any, is at the given location.
+     * This function will pick all drawables that are visible, unless specific
+     * candidate drawable IDs are provided.
      * @param {int} centerX The client x coordinate of the picking location.
      * @param {int} centerY The client y coordinate of the picking location.
      * @param {int} [touchWidth] The client width of the touch event (optional).
      * @param {int} [touchHeight] The client height of the touch event (optional).
-     * @param {Array<int>} [candidateIDs] The Drawable IDs to pick from, otherwise all.
+     * @param {Array<int>} [candidateIDs] The Drawable IDs to pick from, otherwise all visible drawables.
      * @returns {int} The ID of the topmost Drawable under the picking location, or
      * RenderConstants.ID_NONE if there is no Drawable at that location.
      */
-    pick (centerX, centerY, touchWidth, touchHeight, candidateIDs) {
+    pick (centerX, centerY, touchWidth, touchHeight, candidateIDs = this._visibleDrawList) {
         const gl = this._gl;
 
         touchWidth = touchWidth || 1;
         touchHeight = touchHeight || 1;
-        candidateIDs = (candidateIDs || this._drawList).filter(id => {
-            const drawable = this._allDrawables[id];
-            const uniforms = drawable.getUniforms();
-            return drawable.getVisible() && uniforms.u_ghost !== 0;
-        });
```

Comment:
Looks like you can now pick up ghosted sprites, I think this could cause compatibility issues with projects..

Reply:
Yeah - This is definitely a change, but it's a change that I'm wondering if we shouldn't do.  Basically every other collision detect DOES NOT ignore 100% ghosted sprites - What actually uses `pick` (click on stage? etc) I would love to get some "units" written that test these edges and make sure we have the right behavior before moving on with it.  I'd be happy to update this again to default ignore ghosted sprites.

Reply:
I just reverted the behavior here.